pull-request-process.md

```diff
@@ -35,4 +35,4 @@ Fix to [xyz feature] was added to resolve bug referenced in #021, #009. All buil
 
 If you are working on a pull request with many moving parts, we strongly recommend that you break up your request into smaller PRs, rather than one large PR with many lines of code to review. This helps Community Extension Maintainers review and approve your pull requests quickly. 
 
-If you add a [WIP] title to your pull request, or issue a /hold comment, the Camunda GitHub Probot will add the do-not-merge/work in progress label to your issue, and the do-not-merge/hold labels, respectively. While either of these labels is active on your pull request, it will not be considered for merging by a Maintainer or a Reviewer. 
+If you add a [WIP] title to your pull request, or issue a /hold comment, the Camunda GitHub Probot will add the do-not-merge/work in progress label to your issue, and the do-not-merge/hold labels, respectively. While either of these labels is active on your pull request, it will not be considered for merging by a maintainer or a reviewer. 
```

Comment:
Should there be a link attached to [WIP]? Additionally, is there meant to be a backslash in front of "hold"?

Reply:
It looks like these are commands for the CCH Probot, so I added a link clarifying that in this section.